task/02-numbers-tasks.js

```diff
@@ -221,7 +224,9 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
-    throw new Error('Not implemented');
+    let val= new Number (value);
+	if (isNaN(val)) return def
+	else return val
 }
```

Comment:
The function should return `number` primitive type
Number(...) will simply convert the expression into a number primitive value, while new Number(...) will create a wrapper object around the converted number value.